app/controllers/checkins_controller.rb

```diff
@@ -0,0 +1,39 @@
+class CheckinsController < ApplicationController
+  # before_action :set_checkin, only: [:show, :edit, :update, :destroy]
+  protect_from_forgery unless: -> { request.format.json? }
+
+  # GET /checkins
+  # GET /checkins.json
+  def index
+    @checkins = Checkin.all
+  end
+
+  # POST /checkins
+  # POST /checkins.json
+  def create
+    user_id = User.where(uid: checkin_params["fb_user_id"]).first.id
```

Comment:
you could do `User.find_by(uid: checkin_params["fb_user_id"])` and this just returns the first one